src/views/CoCurricularTranscript/Components/CoCurricularTranscriptActivity/coCurricularTranscriptActivity.scss

```diff
@@ -25,7 +25,7 @@
   grid-template-columns: 5% 70% 25%;
   grid-template-rows: auto auto;
   justify-items: stretch;
-  grid-row-gap: 20px;
+  grid-row-gap: 0px;
```

Comment:
Does a 0 pixel gap look different than just not having a gap? (Who knows, weird things go on with these grids)

Reply:
no, it does not. I'll remove that line until we clean up formatting.